grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -198,7 +198,10 @@ export const RotationForm = observer((props: RotationFormProps) => {
     } catch (err) {
       onError(err);
     } finally {
-      setIsOpen(true);
+      // wait until a scroll to the "Rotations" happened
+      setTimeout(() => {
+        setIsOpen(true);
+      }, 100);
```

Comment:
Why do we need to wait for a scroll to happen on Rotations Form? Maybe add a comment to indicate the reason? Seems very hackish